server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/mail/Email.scala

```diff
@@ -82,4 +82,5 @@ case class Email(id: MessageId,
                  attachments: List[EmailBodyPart],
                  headers: List[EmailHeader],
                  bodyValues: Map[PartId, EmailBodyValue],
-                 messageId: Option[List[HeaderMessageId]])
+                 messageId: Option[List[HeaderMessageId]],
+                 inReplyTo: Option[List[HeaderMessageId]])
```

Comment:
`HeaderMessageId` -> `HeaderField`?

Reply:
No, it represents the MessageId type for headers, as described in the JMAP spec.

HeaderXX prefix is here to disambiguate with mailbox/api MessageId

Reply:
Reread the specs, indeed